src/main/menu_main.cpp

```diff
@@ -1532,11 +1533,10 @@ void mainMenuDraw()
             SuperPrint(fmt::format_ne("{0} {1}%", g_mainMenu.loading, progress), 3, MenuX, MenuY);
         }
     }
-#else
-    if(false) {}
+    else
 #endif
     // Main menu
-    else if(MenuMode == MENU_MAIN)
+    if(MenuMode == MENU_MAIN)
```

Comment:
When the "loading" flag is active, you should keep `else`, to don't cause the menu to be drawn, here you would get a crash because, in a moment of loading, the menu gets filled in a separated thread.

Reply:
Okay, I checked the code, now it's fine :fox_face: :+1: 
sorry